src/main/java/io/vertx/amqp/impl/AmqpConnectionImpl.java

```diff
@@ -18,6 +18,7 @@
 import io.vertx.amqp.*;
 import io.vertx.core.*;
 import io.vertx.core.impl.ContextInternal;
+import io.vertx.core.impl.VertxInternal;
```

Comment:
Guessing that this isnt used and should be removed.

Reply:
done